decidim-proposals/config/locales/ca.yml

```diff
@@ -421,6 +419,7 @@ ca:
           error: Hi ha hagut un problema en crear aquests esborranys col·laboratius
           success: S'ha creat un esborrany col·laboratiu.
         edit:
+          attachment_legend: "(Opcional) Afegeix un fitxer adjunt"
```

Comment:
We shouldn't change non-english locales, otherwise we'll have merge conflicts with PRs coming from Crowdin.